flake.nix

```diff
@@ -13,6 +13,8 @@
   inputs.sops-nix.url = "github:Mic92/sops-nix";
   inputs.buildbot-nix.inputs.nixpkgs.follows = "nixpkgs";
   inputs.buildbot-nix.url = "github:nix-community/buildbot-nix";
+  inputs.app-shell.inputs.nixpkgs.follows = "nixpkgs";
+  inputs.app-shell.url = "github:eljamm/app-shell?ref=refactors";
```

Comment:
I'm using my fork of `app-shell` with [some changes](https://github.com/imincik/app-shell/compare/master...eljamm:app-shell:refactors) that makes this work. Specifically, [accepting apps as lists](https://github.com/imincik/app-shell/commit/baeb24bf8db0658f507d97984092ab56b63e5b74) and [making pkgs overridable](https://github.com/imincik/app-shell/commit/51d44e3debd7c31a7712c36673b1f9005e281c27). Which makes me wonder, do we want to maintain this in or out-of-tree?

Reply:
I remember we decided to maintain it in tree but only take the subset that composes `PATH` and possibly `MANTPATH`, i.e. everything that's needed at the CLI level. The other stuff is ecosystem-specific, we don't need that here.

Reply:
Yes, just take required lines of code from app-shell and add them where needed.